src/examples/src/config.tsx

```diff
@@ -103,6 +103,10 @@ import CloseableDialog from './widgets/dialog/CloseableDialog';
 
 `!has('docs')`;
 import testsContext from './tests';
+import ProgressWithChangingValues from './widgets/progress/ProgressWithChangingValues';
```

Comment:
mike can you move these up above the `!has('docs')` line please and alphabetise them like the others? This helps with conflicts later